core-bundle/contao/classes/DataContainer.php

```diff
@@ -703,6 +703,10 @@ protected function row()
 			$objTemplate->fileBrowserTypes = implode(' ', $fileBrowserTypes);
 			$objTemplate->source = $this->strTable . '.' . $this->intId;
 			$objTemplate->readonly = (bool) ($arrAttributes['readonly'] ?? false);
+			$objTemplate->tinyMceLanguage = Backend::getTinyMceLanguage();
+			$objTemplate->theme = Backend::getTheme();
+			$objTemplate->enabled = $GLOBALS['TL_CONFIG']['useRTE'] ?? false;
```

Comment:
This will enable both TinyMCE and ACE, which is probably not what you intended. Do we need something like this?

```php
$objTemplate->enableAce = $GLOBALS['TL_CONFIG']['useCE'] ?? false;
$objTemplate->enableTinyMce = $GLOBALS['TL_CONFIG']['useRTE'] ?? false;
```